### PR TITLE
Tweaks for the T-64 Battle Rifle

### PR DESCRIPTION
**Pull request by joniroq**
## About The Pull Request

This PR allows the T-64 to attach an angled grip. That's it.

## Why It's Good For The Game

The T-64 sits in a weird spot between a marksman rifle and an assault rifle and sees very little use in comparison to its peers. Due to the fact the T-37 can equip an angled grip while the T-64 couldn't made the slower, more damaging T-37 actually faster to wield. Hopefully with this PR it will bring the weapon more in line with being a mid-ground between AR and DMR.

## Changelog
:cl: Joniroq
balance: T-64 battle rifle can now have an angled grip.

/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -275,7 +276,7 @@
 	starting_attachment_types = list(/obj/item/attachable/stock/br,/obj/item/attachable/scope/mini)
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 18,"rail_x" = 17, "rail_y" = 18, "under_x" = 25, "under_y" = 12, "stock_x" = 14, "stock_y" = 10)
 	actions_types = list(/datum/action/item_action/aim_mode)
-	aim_fire_delay = 0.2 SECONDS
+	aim_fire_delay = 0.17 SECONDS
```
Comment (BraveMole) [inline code/modules/projectiles/guns/rifles.dm:279]: must be divisible by 0.05, beause byond

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by SilencedMP5A5** [comment]
the battle rifle is already really good though

for comparison, it's very similar to the lasrifle on overcharge mode

both weapons require good tracking to kill anything though, because your ammo is quite limited

File: code/modules/projectiles/guns/rifles.dm (hunk 1 above)
Comment (joniroq) [reply code/modules/projectiles/guns/rifles.dm:279]: Ahh...crap, I'll revert it then since 0.15 seems a bit too powerful

**Comment by davidchan13** [comment]
Angle grip adds spread so this isn't much of a buff.

**Review by SplinterGP — APPROVED** [review]
Seems fine since angled grip nowadays has actual downside(or almost)

**Pull request merged**